### PR TITLE
Added myself to the Developers

### PR DESCRIPTION
**Pull request by AlexanderHaug**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,4 +3,5 @@
 
 ## Developers
 * Ben Coleman (professor)
-* Willy Brandes (student)
\ No newline at end of file
+* Willy Brandes (student)
```
Comment (bjcoleman) [inline README.md]: These names really should be in alphabetical order.  I know I said I should be first, but I was wrong.  Please reorder.

Comment (AlexanderHaug) [reply README.md]: First Name or Last Name order?

Comment (bjcoleman) [reply README.md]: Last name order, please.

**Comment by bjcoleman** [comment]
Looks good!

**Pull request merged**